tox.ini

```diff
@@ -23,6 +23,17 @@ commands =
     poetry install -q
     poetry run black --check --diff {posargs:.}
 
+[testenv:docs]
+changedir = docs
+deps = sphinx
+whitelist_externals =
+    mkdir
+    rm
+commands=
+    mkdir -p _static
+    rm -rf _build
+    sphinx-build -W -b html -d {envtmpdir}/doctrees .  _build/html
```

Comment:
I had to make the following changes to avoid an `ImportError`:

```diff
--- a/tox.ini
+++ b/tox.ini
@@ -27,12 +27,14 @@ commands =
 changedir = docs
 deps = sphinx
 whitelist_externals =
-    mkdir
+    poetry
     rm
+    mkdir
 commands=
     mkdir -p _static
     rm -rf _build
-    sphinx-build -W -b html -d {envtmpdir}/doctrees .  _build/html
+    poetry install -q
+    poetry run sphinx-build -W -b html -d {envtmpdir}/doctrees .  _build/html
 
 [testenv:licenses]
 commands =
```